### DISCUSSION TITLE
2007/11/13/%E8%A5%BF%E5%AE%89%E7%9A%84%E5%87%BA%E7%A7%9F%E8%BD%A6

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#188] [Announcements]
# 2007/11/13/%E8%A5%BF%E5%AE%89%E7%9A%84%E5%87%BA%E7%A7%9F%E8%BD%A6

上个星期没有更新 blog，我出差到西安去了。

https://blog.qizhen.xyz/2007/11/13/%E8%A5%BF%E5%AE%89%E7%9A%84%E5%87%BA%E7%A7%9F%E8%BD%A6

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Hongbo
2008年09月24日 00:21 编辑
看来广州的2.6相当的贵啊！

回复
Unknown
2008年04月9日 23:11 编辑
呵呵，我是西安人，大学在西安，研究生也在西安，从来没有遇见类似的情况。下次你要来西安的话，有什么事情可先咨询一下我，一定给你帮忙。呵呵，我也是初学labview大军中的一员，看到你blog上的文章觉得很是敬佩，向你学习！

回复
何
2007年12月17日 16:35 编辑
每次用LabVIEW时遇到困难时，我总是上你的Blog逛，每次总有收获，很想对你说声谢谢，今天看到你的这篇Blog，我想我错过了最好的时机，你在西安的时候，我们很可能已经见过面了。那是那段特别的日子里让我感觉最轻松的一次交流，虽然我的表现很差。我很清楚，那只是我的梦想，能够跟你们面对面的交流，我已经很高兴了，非常感谢你们给我的机会！

回复
半夏
2007年11月13日 09:16 编辑
呵呵，挺详细。我去西安时好象没在意那么多，不过倒确实觉得西安的交通挺乱。

回复
